Classes/Form/PostProcess/SaveFormPostProcessorEnhancement.php

```diff
@@ -161,13 +162,43 @@ public function __construct(\TYPO3\CMS\Form\Domain\Model\Element $form, array $t
 	 * @return string HTML message from this processor
 	 */
 	public function process() {
-		$formElements = $this->form->getChildElements();
+        $fieldsToProcessFromTS = array();
+        $fieldsToProcess = array();
+        if( trim($this->typoScript["fieldsToSaveAsTitle"]) ) {
+            $fieldsToProcessFromTS = explode(",", $this->typoScript["fieldsToSaveAsTitle"]);
+            $fieldsToProcess = array_map('trim', $fieldsToProcessFromTS);
```

Comment:
$fieldsToProcess = GeneralUtility::trimExplode(",", $this->typoScript["fieldsToSaveAsTitle"]); so the previous line isnt needed.